app/src/main/AndroidManifest.xml

```diff
@@ -52,6 +52,11 @@
         android:theme="@style/QuranToolBar"
         android:configChanges="keyboardHidden|orientation|screenSize"/>
     <activity
+        android:name=".AdvancedQuranPreferenceActivity"
```

Comment:
IMHO, this should follow existing file naming convention. So, probably, should be named as `QuranAdvancedPreferenceActivity`.